pkl-core/src/test/files/LanguageSnippetTests/output/parser/spacesBetweenDocComments.err

```diff
@@ -0,0 +1,6 @@
+–– Pkl Error ––
+Invalid token at position. Expected class, typealias, method or property.
```

Comment:
Nit: it would be good to customize error messages for doc comments.

Something like:

```
Dangling documentation comment.

x | /// doc comment continuation
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Documentation comments must be attached to modules, classes, typealiases, methods, or properties.
```